docs/support/ghc-version-support.md

```diff
@@ -76,12 +80,43 @@ For example, if 9.2 is the latest major version fully supported by HLS and used
 
 HLS will support all non-legacy major versions of GHC.
 
-### Minor versions
+#### Minor versions
 
 For the latest supported major GHC version we will support at least 2 minor versions.
 
 For the rest of the supported major GHC versions, we will support at least the latest minor version in Stackage LTS (so 1 minor version).
 
+### Extended policy
```

Comment:
I think the base/extended distinction is confusing and you could just drop these headers. It's fairly clear that the later sections talk about exceptions to the rules.

Reply:
Hm, I think we're having a slightly different perception. 

I'm not seeing it as an exception rule that can be applied if someone speaks up about it.

Someone actually updating the supported GHC versions is **required** to check/consided both.

Reply:
Okay, yes, I agree with that. Then maybe we just want to add after line 81: "HLS may continue to support legacy GHC versions, see the section below on 'Supporting legacy GHC versions'" or something?